service/history/hsm/tree_test.go

```diff
@@ -543,13 +563,30 @@ func TestOperationLog_IsDeleted(t *testing.T) {
 	err = l1.DeleteChild(hsm.Key{Type: def1.Type(), ID: "l2"})
 	require.NoError(t, err)
 
-	l2Outputs := l2.Outputs()
-	require.Empty(t, l2Outputs)
+	opLog, err := root.Outputs()
+	require.NoError(t, err)
 
-	// Non-deleted sibling should still have outputs
-	siblingOutputs := l2_sibling.Outputs()
-	require.Len(t, siblingOutputs, 1)
-	require.Equal(t, l2_sibling.Path(), siblingOutputs[0].Path)
+	var foundDelete bool
+	for _, op := range opLog {
+		if delOp, ok := op.(hsm.DeleteOperation); ok {
+			if reflect.DeepEqual(delOp.Path(), l2.Path()) {
```

Comment:
`slices.Equal` also works here.

Reply:
Or replace this whole loop with https://pkg.go.dev/slices#ContainsFunc.

Reply:
Thx for the tip!